### DISCUSSION TITLE
/articles/sharepoint-framework-use-brand-center-custom-fonts/

### DISCUSSION THREAD

**Discussion by giscus** [Voitanos/voitanos.io-comments#312] [Comments]
Developers can use the fonts defined in the new SharePoint brand center in your custom SharePoint Framework (SPFx) with font slots and font tokens - learn how!
https://www.andrewconnell.com/articles/sharepoint-framework-use-brand-center-custom-fonts/

**Comment by StfBauer** [comment]
Seems like the preview has already the day you published your post.
I don't see any indication in my brand center anymore.

**Reply by andrewconnell** [reply]
Sorry... I don't follow... what do you mean by "the preview has already the day?"

**Reply by StfBauer** [reply]
Microsoft removed the "Preview" Label from the Sidebar link and the Brand Center is now GA.

Here it showed the Preview label for a long time:
![image](https://github.com/user-attachments/assets/2a26d440-668e-4c0a-aae9-40213cdea7aa)

In addition the Font is no preview label anymore too.
![image](https://github.com/user-attachments/assets/122ce7bb-91e7-430c-8f5c-6244757c133a)

**Reply by andrewconnell** [reply]
Yup, looks like that JUST happened today...
